galleries/users_explain/colors/colormaps.py

```diff
@@ -230,9 +230,9 @@ def plot_color_gradients(category, cmap_list):
 # These would not be good options for use as perceptual colormaps.
 
 plot_color_gradients('Qualitative',
-                     ['Pastel1', 'Pastel2', 'Paired', 'Accent', 'Dark2',
-                      'Set1', 'Set2', 'Set3', 'tab10', 'tab20', 'tab20b',
-                      'tab20c'])
+                     ['Pastel1', 'Pastel2', 'Paired', 'Accent', 'Accessible',
```

Comment:
```suggestion
                     ['Pastel1', 'Pastel2', 'Paired', 'Accent', 'Okabe-Ito',
```
same w/ the caveat we'd need to discuss names probably